sda-auth/config.go

```diff
@@ -163,16 +165,16 @@ func (c *Config) readConfig() error {
 		log.Printf("Setting log level to '%s'", stringLevel)
 	}
 
-	if viper.GetString("s3Inbox") == "" {
-		return fmt.Errorf("%s not set", "s3Inbox")
-	}
-
 	// no need to check the variables for JWT generation if we won't use it
 	if (cega.ID == "" && cega.Secret == "") && !c.ResignJwt {
 		return nil
 	}
 
-	for _, s := range []string{"jwtIssuer", "JwtPrivateKey", "JwtSignatureAlg"} {
+	if viper.GetString("s3Inbox") == "" {
+		return fmt.Errorf("%s not set", "s3Inbox")
+	}
+
+	for _, s := range []string{"jwtIssuer", "JwtPrivateKey", "JwtSignatureAlg", "c4ghPubKey"} {
```

Comment:
```suggestion
	for _, s := range []string{"c4ghPubKey", "jwtIssuer", "JwtPrivateKey", "JwtSignatureAlg", "s3Inbox"} {
```

Reply:
Is there a reason to remove this block?
```
if viper.GetString("s3Inbox") == "" {
		return fmt.Errorf("%s not set", "s3Inbox")
	}
```
Renamed `c4ghPubKey` to `publicfile`